src/managers/GuildMemberManager.js

```diff
@@ -146,15 +147,21 @@ class GuildMemberManager extends BaseManager {
    * guild.members.prune({ days: 1, reason: 'too many people!' })
    *   .then(pruned => console.log(`I just pruned ${pruned} people!`))
    *   .catch(console.error);
+   * @example
+   * // Include members with a specified role
+   * guild.members.prune({ days: 7, roles: ['657259391652855808'] })
+   *    .then(pruned => console.log(`I just pruned ${pruned} people!`))
+   *    .catch(console.error);
    */
-  prune({ days = 7, dry = false, count = true, reason } = {}) {
+  prune({ days = 7, dry = false, count = true, roles = [], reason } = {}) {
     if (typeof days !== 'number') throw new TypeError('PRUNE_DAYS_TYPE');
     return this.client.api
       .guilds(this.guild.id)
       .prune[dry ? 'get' : 'post']({
         query: {
           days,
           compute_prune_count: count,
+          include_roles: roles,
```

Comment:
```suggestion
          include_roles: roles.map(roleOrId => this.guild.roles.resolveID(roleOrId)),
```

Reply:
Allow me to butt in for a moment, this will NOT work. You need to specify an `include_roles` for EACH role ID..so have fun